### PR TITLE
proj01

### PR DESCRIPTION
**Pull request by chiisalrdytaken**
issue #29 #58 #61 #62 #64 #65 solved

### CODE REVIEW

**Comment by chiisalrdytaken** [comment]
add ant support
modify .gitignore and delete unnecessary files
update README.md

**Comment by chiisalrdytaken** [comment]
fixed small errors in readme.md

File: README.md [hunk 1]
```diff
@@ -93,10 +121,6 @@ The first version of this code pre-dates the use of github for CS56 projects.
 * Send to offline user with an error displayed to the sender
 * Able to delete a contact from the ClientWindow for a specific User and have that change be persistent in the Server
 
-### Bugs
```
Comment (seemantasaha) [inline README.md]: Did you solve all these bugs? If not, do not remove it from readme.

File: README.md [hunk 2]
```diff
@@ -4,23 +4,24 @@ This is a basic "chatroom" program--a barebones implementation of something simi
...
 ===============
 ```
 YES | mastergberry | ericchen94 | A chat application that uses a client and a server
  W14 | jcneally 4pm | ericchen94 | A chat application that uses a client and a server
 ```
 
-## Basic Usage
-
-The server needs to be started first (including pressing "Start server" on the server GUI window).   The following ant target will accomplish that:
+## Basic Usage (updated W18)
```
Comment (seemantasaha) [inline README.md]: Do not include (updated w18)

File: build.xml [hunk 3]
```diff
@@ -24,7 +24,8 @@
 
   <target name="compile" description="compile the code">
     <mkdir dir="build" />
-    <javac srcdir="src" destdir="build" debug="true" debuglevel="lines,source" includeantruntime="false" > 
+    <javac srcdir="src" destdir="build" debug="true" debuglevel="lines,source" includeantruntime="false"
```
Comment (seemantasaha) [inline build.xml]: Make it in one line.

File: w18_proj00.md [hunk 4]
```diff
@@ -1,51 +0,0 @@
-### Chi X, Shengjia Y
```
Comment (seemantasaha) [inline w18_proj00.md]: Why did you remove this file?

Comment (chiisalrdytaken) [reply w18_proj00.md]: move it to historyfiles
